lib/msf/core/web_services/authentication/strategies/api_token.rb

```diff
@@ -48,6 +53,14 @@ def validate_user(user)
         !user.nil? ? {valid: true, code: 0, message: nil} : {valid: false, code: 401, message: "Invalid API token."}
       end
 
+      # Authenticates the API token from a configuration file
```

Comment:
The comment wasn't updated when authentication shifted from a file API token to the environment API token.